consensus/mint-client/src/config.rs

```diff
@@ -111,3 +264,53 @@ fn parse_public_address(b58: &str) -> Result<PublicAddress, String> {
         Err(format!("b58 address '{}' is not a public address", b58))
     }
 }
+
+/// Parses a minting limit and signer set from a string in the format:
+/// mint limit:threshold:keyfile1.pem[:keyfile2.pem...]
+fn parse_mint_config(src: &str) -> Result<(u64, SignerSet<Ed25519Public>), String> {
+    let parts = src.split(':').collect::<Vec<_>>();
+
+    // At the minimum we should have 3 parts: mint limit, signing threshold, one
+    // public key file
+    if parts.len() < 3 {
+        return Err(format!(
+            "mint config '{}' is not in the correct format",
```

Comment:
Optional/Equivalent: specify the correct format
```suggestion
            "mint config '{}' is not in the correct format. Expected format is '<mint_limit>:<signing_threshold>:keyfile1.pem[:keyfile2.pem:...]'",
```